### PR TITLE
Journalist can add images to article 

### PR DESCRIPTION
**Pull request by JaimeCrz**
# [PT Story](https://www.pivotaltracker.com/story/show/171811178)
Co-authored-by: hunter <hmvitous@yahoo.com>
Co-authored-by: karro <karolina.frostare@gmail.com>
Co-authored-by: anish <anish.kanswal@gmail.com>
The above co-authors where also present for the API side of the same feature.

## User Story
```
As a journalist
In order to make my articles more visually appealing
I would like to be able to upload and display images to my articles
```
## Changes proposed in this pull request:
* Imports image feature from Grommit to ArticleList for the displaying images to visitors
* Imports image feature from Grommit to SpecificArticle jsx for the displaying images to visitors
## What I have learned working on this feature:
* Refreshed knowledge on importing from API
## Screenshots (Only if client side story)
image (1)

### CODE REVIEW

File: src/components/ArticleList.jsx [hunk 1]
```diff
@@ -9,12 +9,13 @@ class ArticleList extends Component {
     axios.get("/articles").then(response => {
       this.props.dispatch({
         type: "ARTICLES",
-        payload: { articleList: response.data.articles }
+        payload: { articleList: response.data }
       });
+
```
Comment (hmvitous) [inline src/components/ArticleList.jsx]: this space may be an extra one that can be deleted

**Review by hmvitous — CHANGES_REQUESTED** [review]
Once the extra space is gone or its explained to me if it needs to stay then its all good to merge.

File: src/components/ArticleList.jsx (hunk 1 above)
Comment (JaimeCrz) [reply src/components/ArticleList.jsx]: fixed

**Review by hmvitous — APPROVED** [review]

**Comment by kfrostare** [comment]
I have no additional comments on this PR /Kf

**Pull request merged**